CondFormats/PPSObjects/test/BuildFile.xml

```diff
@@ -10,6 +10,6 @@
   <use name="CondFormats/PPSObjects"/>
 </bin>
 
-<bin file="testSerializationPPSAlignmentConfig.cc">
```

Comment:
The PPSAlignmentConfig class is left in the code, or I did not notice the delete line? In the first case, why you remove its test ( and the above entries in the FETCH/IMPORT/PAYLOAD_2XML macros?

Reply:
You are right, I did not remove the `PPSAlignmentConfig` class. Even though it's obsolete, as far as I understand, I can't remove it (and the record) since it is a conditions format (with some payloads already in the Conditions DB). Because it won't be used any more, I removed the test, the write/retrieve modules and the entries in `CondCore/Utilities` (772b3e45b8734af9838f345bcd8d3affcd61ef5e), but I can revert these changes. What do you suggest?

Reply:
If no workflow code is consuming the old payloads, in principle we could remove it. However, it is a choice that presents some risks... I would propose to consider its deletion in a future PR. In this one, please revert the changes related to the old CondFormats (tests and FETC/IMPORT/PAYLOAD_2XML). Thanks.

Reply:
Done - now the files/lines of code related to the PPSAlignmentConfig class are not modified in this PR.